src/modules/pulumi.rs

```diff
@@ -0,0 +1,295 @@
+#![warn(missing_docs)]
+use sha1::{Digest, Sha1};
```

Comment:
Just want to say in advance: I'm not at all familiar with pulumi, so forgive me if some of these questions are naive.